tests/integration/test_script_runner.py

```diff
@@ -15,8 +15,17 @@
         "data_context_dir": "tests/integration/fixtures/yellow_trip_data_pandas_fixture/great_expectations",
         "data_dir": "tests/test_sets/taxi_yellow_trip_data_samples",
         "user_flow_script": "tests/integration/fixtures/yellow_trip_data_pandas_fixture/two_batch_requests_two_validators.py",
-        "expected_stderrs": "",
-        "expected_stdouts": "",
+        # "expected_stderrs": "",
```

Comment:
Just a suggestion: If these are not used it may be better to remove rather than comment them out.